controllers/operator.go

```diff
@@ -271,6 +314,12 @@ func (o *operator) syncDeploys(managedDeploys appsv1.DeploymentList) {
 	consumerStatus.WithLabelValues(name, "outdated").Set(float64(*status.Outdated))
 	consumerStatus.WithLabelValues(name, "expected").Set(float64(*status.Expected))
 	consumerStatus.WithLabelValues(name, "missing").Set(float64(*status.Missing))
+	consumerStatus.WithLabelValues(name, "redundant").Set(float64(*status.Redundant))
+
+	if len(o.missingIds) > 0 {
```

Comment:
Is there a point for this log message?

Reply:
I guess not, I initially added several of those, but then removed. removing this as well.